pages/common/flake8.md

```diff
@@ -0,0 +1,20 @@
+# flake8
+
+> Tool to check the style and quality of Python code.
+> More information: <https://flake8.pycqa.org/>.
+
+- Lint a file or directory recursively:
+
+`flake8 {{path/to/file_or_directory}}`
+
+- Show the line on which each error occurred:
+
+`flake8 --show-source {{path/to/file_or_directory}}`
+
+- Ignore a list of rules. (All available rules can be found at flake8rules.com):
+
+`flake8 --ignore {{rule1,rule2}} {{path/to/file_or_directory}}`
+
+- Exclude files matching the given globs or substrings:
+
```

Comment:
```suggestion
- Lint a file or directory recursively and how the line on which each error occurred:

`flake8 --show-source {{path/to/file_or_directory}}`

- Lint a file or directory recursively and ignore a list of rules. (All available rules can be found at flake8rules.com):

`flake8 --ignore {{rule1,rule2}} {{path/to/file_or_directory}}`

- Lint a file or directory recursively but exclude files matching the given globs or substrings:

```

Reply:
hmm, that doesn't show changes very clearly, however I have only added the first little sentence to each since I like complete clarity.